src/crypto/openssl/openssl_wrappers.h

```diff
@@ -255,16 +255,34 @@ namespace ccf::crypto
       {}
     };
 
+    static const char pem_prefix[] = "-----BEGIN CERTIFICATE-----\n";
+    static constexpr size_t pem_prefix_len = sizeof(pem_prefix) - 1;
```

Comment:
Nit. Could we just remove `\n` from `pem_prefix` and turn `sizeof() - 1` `-> `sizeof()`? Should work? Or replace with `std::string` + `std::string::data()`?

Reply:
The -1 is for the null termination, not for the "\n". Don't think we can get a constexpr size() on a std::string, especially on the clang-11 toolchain but will give it a try.

Reply:
> The -1 is for the null termination
My bad. I totally forgot compiler adds an extra `\0' to `char[]`.

Reply:
Basically, I confused that with `strlen`

Reply:
No need to change unless you wish to, but probably worth adding a comment why `-1`